src/regs/mair_el1.rs

```diff
@@ -322,105 +322,105 @@ register_bitfields! {u64,
         ],
 
         /// Attribute 1
-        Attr1_HIGH OFFSET(12) NUMBITS(4) [
+        Attr1_Normal_Outer OFFSET(12) NUMBITS(4) [
             Device = 0b0000,
 
-            Memory_OuterWriteThrough_Transient_WriteAlloc = 0b0001,
-            Memory_OuterWriteThrough_Transient_ReadAlloc = 0b0010,
-            Memory_OuterWriteThrough_Transient_ReadAlloc_WriteAlloc = 0b0011,
+            WriteThrough_Transient_WAlloc = 0b0001,
```

Comment:
Aarch64 ignores transient on Outer so i didn't code these. Not sure if that's the right answer - they are defined.